six/modules/c++/cphd03/source/VBM.cpp

```diff
@@ -321,7 +321,7 @@ VBM::VBM(size_t numChannels,
          bool srpTimeEnabled,
          bool tropoSrpEnabled,
          bool ampSFEnabled,
-         DomainType domainType,
+         cphd::DomainType domainType,
          const std::vector<const void*>& data) :
```

Comment:
This constructor seems to allow VBM data to be set without byte swapping the data to big endian?
Is the constructor assuming the data being passed in as already big endian? If so should I add that to the comments?

Reply:
It looks like the constructor populates the individual parameter values, so it expects the data to be in native-Endian format.